src/main/js/store/actions/distributionConfigs.js

```diff
@@ -241,3 +281,24 @@ export function checkDescriptorForGlobalConfig(errorFieldName, descriptorName) {
         }).catch(console.error);
     };
 }
+
+export function validateDistributionJob(config) {
+    return (dispatch, getState) => {
+        dispatch(validatingJob());
+        const { csrfToken } = getState().session;
+        const request = ConfigRequestBuilder.createValidateRequest(ConfigRequestBuilder.JOB_API_URL, csrfToken, config);
+        request.then((response) => {
+            response.json()
+                .then((responseData) => {
+                    const handler = createErrorHandler(DISTRIBUTION_JOB_VALIDATE_ERROR, () => validateJobError(responseData));
+                    if (responseData.errors && !Object.keys(responseData.errors).length) {
+                        dispatch(validatedJob());
```

Comment:
It seems like we are checking does the field "errors" have any values and if it does then do the dispatch but we are not passing the errors along. Is this expected? Are the errors that we checked for being ignored/lost?

Reply:
Notice the `!` (NOT) operator prefixing the second condition.

Reply:
yes the errors are passed in the else case.  the dispatch of the validatedJob() sets the state that signifies the data is valid. The else called handler(400) meaning bad request so the  default handler will call validateJobError which takes the responseData that contains the message and the errors and set the status to ERROR so a save will never be called.